vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-of your application with the backend. This BFF Code Pattern will help
-you build that component according to IBM’s best practices.
-
-# PDA Micro Frontends
-
-This section provides an overview of each of the micro frontends (MFE)
-that are available as part of the Entando PDA plugin. In cases below
-where there are details specific to the PAM implementation they are
-provided for that MFE. When extending the integration layer to other
-engines or custom implementations it will be up to the integrating team
-to define behavior and datasource for each MFE.
-
-## Task List
-
-The task list MFE provides a user with a list of visible tasks that are
-either assigned to that user or that are potentially ownable by that
-user. In the default implementation the visible tasks are limited to a
-single process instance. At configuration time the application designer
-is given the option to select a set of columns that will be visible in
-the task list for that page.
-
-### PAM Implementation
-
-The default PAM implementation makes the top level task fields available
-in the task list for selection. It is possible to fetch task and process
-variables for rendering in the task list but these values are excluded
-by default due to the additional API call overhead required.
-
-## Task Details
-
-The task details MFE renders detailed information about a given task.
-The task details widget is intended to give the final user processing a
-task the information necessary to complete the task at hand. The task
-details are rendered in a read only grid. See the styling section below
-for changing and customizing the layout.
-
-### PAM Implementation
-
-The PAM integration renders task variables in the task details widget.
+### Installation Steps
 
-## Task Comments
+1. Apply the definitions for the bundle that comprises the PDA plugin. You'll need to adjust the `-n entando` option in each command to match your namespace or project.
+```
+ent bundler from-git -r https://github.com/entando-samples/entando-process-driven-plugin-bundle.git -d | ent kubectl apply -n entando -f -
+```
 
-The task comments MFE gives the user the ability to view notes attached
-to a given task and to add new notes to a task.
+2. Log into your App Builder instance.
 
-### PAM Implementation
+3. Select `Repository` from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot below.
 
-The PAM implementation reads and publishes notes to the comments
-endpoint
+4. Select `Install` to install the bundle. An installation can take several minutes while the application downloads the Linux images for the microservices and installs the related assets.
 
-## Task Forms
+5. Setup permissions to configure the service by following the [Initial Deploy instructions](https://github.com/entando/entando-process-driven-plugin#initial-deploy). Note that Step 1 in this sequence applies to Entando 6.3.0 only.
```

Comment:
We should include the actual steps here. I think we discussed avoiding linking to the source repos for this kind of thing since we won't be able to control the style/language and it could change per version.

Reply:
!!! sorry - i thought we decided to link to this! but agreed on source repos; will change

Reply:
@nshaw should the note about 6.3.0 still be included? is there a large user base for that rev?

Reply:
Per our conversation, nope. Just the 6.3.2 section.